libs/service/api_service.cpp

```diff
@@ -291,6 +291,13 @@ ebpf_verify_and_load_program(
         return EBPF_INVALID_ARGUMENT;
     }
 
+    // Set the default execution type to JIT. This will eventually
+    // be decided by a system-wide policy. Issue #288 to configure
```

Comment:
```suggestion
    // be decided by a system-wide policy. TODO(Issue #288): Configure
```
Helpful to use the keyword TODO to make searching for all TODOs easier